src/core/solutions/cosine_mixture.cpp

```diff
@@ -0,0 +1,48 @@
+#include <math.h>
+#include <core/solutions/cosine_mixture.h>
+
+namespace dnn_opt
+{
+namespace core
+{
+namespace solution
+{
+    cosine_mixture* cosine_mixture::make(generator* generator, unsigned int size)
+    {
+        auto* result = new cosine_mixture(generator, size);
+        
+        result->init();
+        
+        return result;
+    }
+    float cosine_mixture::calculate_fitness()
+    {
+        float result1 = 0;
+        float result2 = 0;
+        float resultf = 0;
+        float* params = get_params();
+        
+        solution::calculate_fitness();
+        
+        for(int i = 0;i < size(); i++){

```

Comment:
Leave a space after the first semicolon. The same happens in a few lines below...